daemon/oci_windows.go

```diff
@@ -355,6 +320,48 @@ func (daemon *Daemon) createSpecLinuxFields(c *container.Container, s *specs.Spe
 	return nil
 }
 
+func setWindowsResourceLimits(c *container.Container, s *specs.Spec, isHyperV bool) {
```

Comment:
Nit - should this just be called `SetResourcesInSpec` to be more obvious what it's doing?